modules/cluster-logging-cpu-memory.adoc

```diff
@@ -60,16 +60,14 @@ spec:
             cpu: 100m
             memory: 100Mi
       replicas: 2
-  collection:
-    logs:
-      type: "fluentd"
-      fluentd:
-        resources: <3>
-          limits:
-            memory: 736Mi
-          requests:
-            cpu: 200m
-            memory: 736Mi
+collection:
+    resources: <3>
+      limits:
+        memory: 736Mi
+      requests:
+        cpu: 200m
+        memory: 736Mi
+    type: fluentd
```

Comment:
Should add `observability/logging/log_visualization/logging-kibana.adoc` to the assemblies list at some point